src/Abc.Zebus.Testing/TestBus.cs

```diff
@@ -20,11 +21,18 @@ public class TestBus : IBus
         private readonly List<ICommand> _commands = new List<ICommand>();
         private readonly HashSet<Subscription> _subscriptions = new HashSet<Subscription>();
 
-        public TestBus()
+        public TestBus(IMessageSerializer messageSerializer)
```

Comment:
I'd prefer to have just an additional property instead of a constructor:

```C#
public IMessageSerailizer Serializer { get; set; } = new MessageSerializer();
```